unit-10/javascrip/main.js

```diff
@@ -0,0 +1,104 @@
+// lesson 1
+function printMultiplicationTable(number) {
+  if (typeof(number) !== 'number') {
+    return 'Data nhập vào phải là số'
+  }
+  for (let i = 1; i <= 10; i++) {
+    console.log(number + ' x ' + i + ' = '+ number * i )
```

Comment:
Dùng string template đi
```suggestion
    console.log(`${number} x  ${i}  = ${number * i}` )
```